pack/entrypoint.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+set -e
+
+# Create necessary directories for s6-overlay
+
+mkdir -p /etc/s6-overlay/s6-rc.d
+mkdir -p /etc/s6-overlay/s6-rc.d/user/contents.d
+
+
+########################################
+# GPUStack s6 Overlay Service
+########################################
+
+mkdir -p /etc/s6-overlay/s6-rc.d/gpustack
+mkdir -p /etc/s6-overlay/s6-rc.d/gpustack/dependencies.d
+
+# Define service type
+cat > /etc/s6-overlay/s6-rc.d/gpustack/type <<EOF
+longrun
+EOF
+
+# Define service run script
+cat > /etc/s6-overlay/s6-rc.d/gpustack/run <<EOF
+#!/command/with-contenv /bin/bash
+# ================================
+# GPUStack longrun service
+# ================================
+
+source /etc/profile
+exec gpustack start $@
+EOF
+chmod +x /etc/s6-overlay/s6-rc.d/gpustack/run
+
+# Define user contents
+touch /etc/s6-overlay/s6-rc.d/user/contents.d/gpustack
+
+########################################
+# Postgres s6 Overlay Service
+########################################
+
+if [[ " $* " == *" --database-url "* ]] || [ -n "${DATABASE_URL}" ]; then
```

Comment:
Can we use gpustack to determine whether bring up postgres or not? If the database-url is configured in `config.yaml`, it can't be detected here.

Reply:
Now gpustack depends on postgres, If they, in turn, rely on gpustack, the dependency will be more complicated. Checking the database-url from command, environment, config file to determine whether to start the postgres.